template/lambda-sqs-worker-cdk/shared/context-types.ts

```diff
@@ -1,30 +1,21 @@
-/* eslint-disable new-cap */
-import * as t from 'runtypes';
+import { z } from 'zod';
 
-export const stageContext = t.Union(t.Literal('dev'), t.Literal('prod'));
-export type StageContext = t.Static<typeof stageContext>;
+export const stageContextSchema = z.enum(['dev', 'prod']);
```

Comment:
It looks like we have some inconsistency on whether these schemata should be lower (`stageContextSchema`) or upper (`StageContextSchema`) camel-cased. Can we try to align on one before merging this PR?

Reply:
Good catch👍! I have made them the same(upper camel-cased) as other templates.